src/Microsoft.SqlTools.Hosting.v2/Properties/AssemblyInfo.cs

```diff
@@ -42,5 +42,4 @@
 [assembly: AssemblyInformationalVersion("1.0.0.0")]
 
 [assembly: InternalsVisibleTo("Microsoft.SqlTools.Hosting.UnitTests")]
-[assembly: InternalsVisibleTo("Microsoft.SqlTools.CoreServices")]
```

Comment:
Is there a team guidance for appropriate use of InternalsVisibleTo? It got way out of hand in previous projects.